routers/users.js

```diff
@@ -12,7 +12,23 @@ router.get(path + '/', (req, res) => {
 })
 
 router.post(path + '/', (req, res) => {
-
+  const response = {
+    success: false,
+    message: ''
+  }
+  const userData = req.body
+  // TODO: encrypt passwords
+  db.teams.createTeam(userData.name, userData.email, userData.password, true).then(() => {
```

Comment:
You probably should confirm that all of these variables exist before you pass them to the function. Maybe check to see if any are missing and return 400 if they are, with a message explaining what is missing. Also, maybe change 'name' to 'username' while you are at it.

Reply:
These are getting checked inside of the createTeam function, so they're being checked for existing, just not in the route.

Reply:
I know it seems like over checking, but I think it will make the response of the code more clear when someone wants to look at it later. The createTeam function checks it to be sure that there are no errors when executing in the database, this endpoint function would check it because if they don't exist then the user has not provided all the required fields and needs to be notified.

Either way, you should probably respond to the user and note what fields are missing